sdks/typescript/biome.json

```
@@ -0,0 +1,55 @@
+        "useSortedClasses": "error"
+      },
+      "style": {
+        "noNonNullAssertion": "warn",
+        "useImportType": "error"
+      },
+      "suspicious": {
+        "noExplicitAny": "warn"
+      }
+    }
+  },
+  "assist": {
```

Biome (1.9.4)

[error] Found an unknown key `assist`.

Known keys:


(deserialize)